### PR TITLE
Revert merge mistake of deleting initialization of CoilDesWaterDeltaT

### PR DESCRIPTION
**Pull request by rraustad**
Pull request overview
---------------------
Corrects merge mistake made 4 days ago [here](https://github.com/NREL/EnergyPlus/commit/fb7543800d9a4f53560fdb3c9e4d9209c4fe1a1f#diff-de781ad7ec80e690e175811f4da40a5aL1282). Any branches created after this time should pull develop to correct the uninitialized variable warning.
Addresses #6849.

### Work Checklist
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [x] Title of PR should be user-synopsis style (clearly understandable in a standalone changelog context)
 - [x] At least one of the following appropriate labels must be added to this PR to be consumed into the changelog:
   - Defect: This pull request repairs a github defect issue.  The github issue should be referenced in the PR description
   - Refactoring: This pull request includes code changes that don't change the functionality of the program, just perform refactoring
   - NewFeature: This pull request includes code to add a new feature to EnergyPlus
   - Performance: This pull request includes code changes that are directed at improving the runtime performance of EnergyPlus
   - DoNoPublish: This pull request includes changes that shouldn't be included in the changelog

### Review Checklist
This will not be exhaustively relevant to every PR.
 - [x] Code style (parentheses padding, variable names)
 - [x] Functional code review (it has to work!)
 - [x] If defect, results of running current develop vs this branch should exhibit the fix
 - [x] CI status: all green or justified
 - [x] Performance: CI Linux results include performance check -- verify this
 - [x] Unit Test(s) - should fix exisiting unit test HeatRecoveryHXOnMainBranch_SimHeatRecoveryTest which was failing on Windows debug locally

### CODE REVIEW

**Comment by rraustad** [comment]
@mjwitte @Myoldmopar merge at will.

File: src/EnergyPlus/ReportSizingManager.cc [hunk 1]
```diff
@@ -1279,6 +1279,7 @@ namespace ReportSizingManager {
                         }
                     }
                 } else if (SizingType == CoolingWaterflowSizing) {
+                    CoilDesWaterDeltaT = DataWaterCoilSizCoolDeltaT;
```
Comment (rraustad) [inline src/EnergyPlus/ReportSizingManager.cc:1282]: This is the only line I changed. Looks like clang format can't make up it's mind from one commit to the next.

Comment (mjwitte) [reply src/EnergyPlus/ReportSizingManager.cc:1282]: Must be some slight differences in how the clang format rules are applied? Curious - are you using the recent VS with built-in clang formatting, or the plugin with an earlier VS?

Comment (rraustad) [reply src/EnergyPlus/ReportSizingManager.cc:1282]: I'm using Community 2017 at home (this commit) and Professional 2017 at work. I see Enable ClangFormat support under Tools/Options/Text Editor/CC++/Formatting (box checked). I can't say that I know how to set up clang formatting or even if I have to do something but it appears to be working.

**Comment by rraustad** [comment]
Results look good. Plant temps changed by 6.67 C and warnings are now gone in OutdoorAirUnitwithAirloopHVAC example file.

**Comment by mjwitte** [comment]
Ran unit tests locally here on Windows, not going to wait for windows CI to report in.

**Pull request merged**